src/pb/publisher_options.rs

```diff
@@ -2,12 +2,15 @@
 // SPDX-FileCopyrightText: © Contributors to the iceoryx-rs project
 // SPDX-FileContributor: Mathias Kraus
 
+use crate::ConsumerTooSlowPolicy;
+
 use std::marker::PhantomData;
 
 pub(super) struct PublisherOptions {
     pub history_capacity: u64,
     pub node_name: String,
     pub offer_on_create: bool,
+    pub subscriber_too_slow_policy: ConsumerTooSlowPolicy,
     _phantom: PhantomData<()>,
```

Comment:
For what is `PhantomData` required?

Reply:
This is required for the generic parameter `T`. Since no other member is dependent on the, some phantom data is required. This does not affect the size of the struct. It is just a marker. See also https://doc.rust-lang.org/std/marker/struct.PhantomData.html